commands/bootstrap/__tests__/__fixtures__/with-contents/lerna.json

```diff
@@ -0,0 +1,4 @@
+{
+  "ci": false,
```

Comment:
I don't see any bootstrap tests using this fixture? In any case, I think you can skip it (and remove this fixture tree), because aside from the `--contents` flag (thank you!), I don't think bootstarp should care about what is an implementation detail of the symlinking lib.

Reply:
Left a comment, this was a mistake, for some reason I thought I needed the fixture for the test 🤷‍♂ Thanks!